Mathlib/Probability/Kernel/Disintegration/Basic.lean

```diff
@@ -0,0 +1,155 @@
+/-
+Copyright (c) 2024 Yaël Dillies, Kin Yau James Wong. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Kin Yau James Wong
```

Comment:
```suggestion
Authors: Yaël Dillies, Kin Yau James Wong, Rémy Degenne
```
Since a non-negligible part of that code is moved from the other file (although slightly changed).